stream/chunk_writer.js

```diff
@@ -21,10 +21,19 @@
 'use strict';
 
 var inherits = require('util').inherits;
+var inspect = require('util').inspect;
 var Transform = require('stream').Transform;
+var WrappedError = require('error/wrapped');
 
 var toBufferTuple = require('../interface').toBufferTuple;
 
+var ChunkWriteError = WrappedError({
+    type: 'chunk-write',
```

Comment:
i recommend namespacing them. `bufrw` errors will flow up through tchannel.

Within tchannel its nice to know if an error came from tchannel or bufrw. Further more within `tchannel-as-thrift` its nice to know whether it came from tchannel-as-thrift, thriftrw, tchannel or bufrw.